src/en/proposals/revolution-rework.md

```diff
@@ -0,0 +1,166 @@
+>Hereinafter, a loyalist is a character with a mindshield implant. In the points system, a loyalist and a head are different people.
+
+**FOR WHAT:** Diversify the method of recruitment, allow simple revolutionaries to recruit, and make a reference to 1984.
+
+### 1.5 Rebel Flag
+- There is another free item in headrev uplink _(1.1)_ - rebel flag, which **can be placed** in different departments.
+- Used exclusively for the points system _(3.2)_.
+- If the department contains **[number of loyalists / 5]**, it is **automatically removed** after a minute.
+- Normal crew can remove it manually activating the process for 1 minute (can't move). Can't be moved.
+- **Can't be staged after** the evacuation shuttle has been dispatched.
+
+**FOR WHAT:** to get rid of stealthy revolutions, diversify the strategic part of the mode, and supplement the points system.
+
+### 1.6 Alert
+> Automatic station notification:
+“Attention! According to automatic internal sensors, there is crew misconduct on the station. There is a threat of a full-scale mutiny. 
+We ask the station heads to take measures according to the SOP and common sense. If necessary, send a fax to the CC regarding the current situation.”
+
+#### This announcement comes after the fact:
+- **over 40%** of the crew have been turned to the revolutionary side 
+- **over 80%** of the rebels have exposed their holographic armband
+- **more than 2 heads or 50%** of the loyal crew are in a neutralized state within 5 minutes
+- **more than 30%** of the crew has dangerous items (homemade, contraband, explosives) in their inventory
+- **there are 3 or more rebel flags** on the station
+
+**One minute before the announcement itself, a message will be given to all headrevs that they have been revealed.**
+
+**FOR WHAT:** to get rid of covert revolutions. 
+
+### 1.7 Headrev
+**When a headrev dies**, revolutionaries are **not deconverted**, but there will be no new headrev. Calculation of the number of hedrevs is the same as before.
+
+### 1.8 Recruitment
+- Upon headrev ability, the victim will be presented with a **dialog box** with the options "YES" (join the revolution and convert) or "NO" (refuse the revolution)
+- In order to recruit a person it is necessary for them **to be free and not implanted with a mindshield**. The outcome of the recruitment process is a handshake. 
+- **[1 converted = 1 RTK]** to the revolutionary's uplink _(1.1)_.
+- The ability has a 1.5 minute **cooldown**.
+- Only the headrev himself or Goldstein's book _(1.4)_ can recruit.
+
+> Will create as a partial necessity for meetings and speeches. Dialogue (RP process) - as a recruitment tool.
+
+**FOR WHAT:** Completely abandon the violent conversion mechanic, diversify the RP process.
+
+## 3. GREENTEXT
+Due to the fact that normal programmatic determination of winners in the mode requires a large and intricate game code/mechanics, everything can be reduced to two simple things: 
```

Comment:
I don't think relying on programatic win conditions is a good idea. It's alot better to make the objective something simple that players can directly understand (like capture command).

Reply:
I've been thinking about this and as has been written - it's hard to find a specific goal for a regime to win, like the usual traitors. Of course, you could try and create a fancy goal system for revolutionaries, but I think it would be difficult both from the gameplay design and programming side.